src/translations/index.js

```diff
@@ -2,9 +2,10 @@ import * as en from "./en.json";
 import * as fr from "./fr.json";
 import * as es from "./es.json";
 import * as hu from "./hu.json";
+import * as pt from "./pt.json";
 
 export const defaultLocale = "en";
-const translations = { en, fr, es, hu };
+const translations = { en, fr, es, hu, pt };
```

Comment:
Are Russian translations connected? I don't see reference here or in imports 🤷‍♂️